src/components/Header/Logo.js

```diff
@@ -2,18 +2,19 @@ import React from 'react';
 import { Link as LinkBase } from 'react-router-dom';
 import styled from 'styled-components';
 
-import { colorBlack, spacingSmall, borderWidthThin, fontSmall } from '../../styles/designTokens';
+import { colorBlack, spacingSmall, borderWidthThin } from '../../styles/designTokens';
 import { Logo as LogoIcon } from '../Icons';
 
 const Link = styled(LinkBase)`
   display: flex;
   text-decoration: none;
   align-items: center;
   height: 100%;
+  cursor: pointer;
```

Comment:
Shouldn't it be pointer by default?

Reply:
For some reason without it cursor sometimes was changing to default one when hovering over tagline.